### PR TITLE
Updated release notes for Keycloak 20

### LINKED ISSUE #1711: Review release notes for Keycloak 20.0.0
### Description

Review and make sure all relevant features/enhancements are included in the release notes:

https://github.com/issues?q=user%3Akeycloak+milestone%3A20.0.0+is%3Aclosed+label%3Arelease-notes

### PR DESCRIPTION
**Pull request by stianst**
Closes #1711

### CODE REVIEW

File: release_notes/topics/20_0_0.adoc [hunk 1]
```diff
@@ -19,24 +43,56 @@ In this release, we are making important changes to `kc.bat` to give the same ex
...
+
+= Custom Identity Providers can now set an icon for the provider
+
+A custom identity provider can now set the icon used on the login pages. Thanks to https://github.com/klausbetz[Klaus Betz],
+who happens also to maintain
+https://github.com/klausbetz/apple-identity-provider-keycloak[an extension to Keycloak to support log in with AppleID].
+
+= Open Banking UK support
+
+Keycloak is now compliant with Open Banking UK. Details in the link:{adapterguide_link}#_open-banking-uk[{adapterguide_name}].
```
Comment (mposolda) [inline release_notes/topics/20_0_0.adoc]: This link with the details is not available in this PR. It requires additional changes in the file "fapi-support.adoc", which I've included in the other PR: https://github.com/keycloak/keycloak-documentation/pull/1709/files . So either this change in the "fapi-support.adoc" should be added as well to this PR or the sentence : `Details in the link:{adapterguide_link}#_open-banking-uk[{adapterguide_name}]` should be removed.

**Review by mposolda — CHANGES_REQUESTED** [review]
Looks good, but there is one detail regarding Open Banking inline.

File: release_notes/topics/20_0_0.adoc (hunk 1 above)
Comment (stianst) [inline release_notes/topics/20_0_0.adoc]:
```suggestion
Keycloak is now compliant with Open Banking UK.
```

Comment (stianst) [reply release_notes/topics/20_0_0.adoc]: I dropped the link, and then you can add it back again in your PR with the docs update if you want? Does that work?

**Review by mposolda — APPROVED** [review]

**Comment by mposolda** [comment]
@stianst Approving, Thanks for the update

**Pull request merged**